embedding/README.md

```diff
@@ -0,0 +1,61 @@
+# embedding-api
+
+## How run servis
+
+### GPU option
+
+1. Instal git lfs
+2. NVIDIA Docker Toolkit
+3. Add shh key to huggingface  account
+4. run this in ChatKNML/ :
+
+```sh
+cd embedding_models
+git clone git@hf.co:intfloat/e5-large-v2
+```
```

Comment:
**issue:** DO NOT do it like this. either use submodules or subtrees. cloning on your own into folder inside repo is just dumb.

Reply:
Ok. I'll do it as submodule